### PR TITLE
docs: Why GeoJSONs are not compressed TDE-1263

### PR DESCRIPTION
**Pull request by l0b0**
### Motivation

Explain to end users why GeoJSON files are not compressed in our public buckets.

### Modifications

Add file with documentation

### Verification

N/A

### CODE REVIEW

File: docs/GeoJSON-compression.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## Arguments
+
+Pro compression:
+
+- Saves money on storage
+- Saves time and money on transfer of data
+
+Contra compression:
+
+- Some tools do not seamlessly decompress files
```
Comment (blacha) [inline docs/GeoJSON-compression.md:16]: could we expand this, what options we looked at why they are not supported.

Things that would be good to document:

- S3 public https://.. endpoints do not compress objects automatically
- We could use S3's `content-encoding` header gzip, but the `aws` cli and SDKs do not automatically decompress `content-encoding` and its not a common pattern for people to use

File: docs/GeoJSON-compression.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# GeoJSON compression
```
Comment (blacha) [inline docs/GeoJSON-compression.md:1]: we should expand on why and what files we want to compress.

We have millions of JSON files eg STAC and some asset files that are highly compressible, its cheaper to store and serve as compressed files.

Do we have context on the size savings? for example we may have 20MB of capture-assets but then 200GB of tiff files which could also lead into the reasoning for not caring about if these are compressed.

File: docs/GeoJSON-compression.md (hunk 1 above)
Comment (l0b0) [reply docs/GeoJSON-compression.md:16]: The first assertion sounds like a reason *pro* us doing compression manually.

Comment (blacha) [reply docs/GeoJSON-compression.md:16]: The first assertion is a massive pro for storing them compressed and its quite common to store them compressed when only serving files via HTTP.

But due to the overhead of second assertion it makes storing them compressed very painful.

**Review by paulfouquet — APPROVED** [review]

**Pull request merged**